features/checkout/paying_for_order/preventing_to_change_payment_method_of_already_cancelled_order.feature

```diff
@@ -0,0 +1,24 @@
+@paying_for_order
+Feature: Preventing to change payment method of already cancelled order
+    In order to perform only valid order operations
+    As a Customer
+    I should be prevented from changing payment method of the cancelled order
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Angel T-Shirt"
+        And the store ships everywhere for Free
+        And the store allows paying with "Cash on Delivery"
+        And the store also allows paying with "Bank Transfer"
+        And I am a logged in customer
+        And I placed an order "#00000666"
+        And I bought a single "Angel T-Shirt"
+        And I addressed it to "Lucifer Morningstar", "Seaside Fwy", "90802" "Los Angeles" in the "United States" with identical billing address
+        And I chose "Free" shipping method with "Cash on Delivery" payment
+
+    @ui @api
+    Scenario: Being prevented to change payment method of cancelled order
+        Given I want to change this order's payment method
+        And this order was cancelled
```

Comment:
Now we have only one scenario so those `givens` should be placed in the `Background` block